src/styles/getModalStyles.js

```diff
@@ -19,6 +19,37 @@ export default (type, windowDimensions, popoverAnchorPosition = {}) => {
     let shouldAddTopSafeAreaPadding = false;
 
     switch (type) {
+        case CONST.MODAL.MODAL_TYPE.CONFIRM:
+            // A confirm modal is one that has a visible backdrop
+            // and can be dismissed by clicking outside of the modal.
+            modalStyle = {
+                ...modalStyle,
+                ...{
+                    alignItems: 'center',
+                },
+            };
+            modalContainerStyle = {
+                // Shadow Styles
+                shadowColor: colors.black,
+                shadowOffset: {
+                    width: 0,
+                    height: 0,
+                },
+                shadowOpacity: 0.1,
+                shadowRadius: 5,
+
+                borderRadius: 12,
+                overflow: 'hidden',
+                width: isSmallScreenWidth ? '90%' : windowWidth * 0.3,
```

Comment:
I think we want to have a fixed width on all non-small screens, so I think just using 375 (the same width as the sidebar) works. 

For small screens, I think it makes sense to repeat the bottomDocked modal pattern here where it would look something like:
![image](https://user-images.githubusercontent.com/2319350/111968347-56a3d780-8af9-11eb-87c4-bc176f5f1128.png)

cc @roryabraham

Reply:
@shawnborton Sure. I'll update the changes.